examples/pytorch_distributed_simple.py

```diff
@@ -0,0 +1,184 @@
+"""
+Optuna example that optimizes multi-layer perceptrons using PyTorch distributed.
+
+In this example, we optimize the validation accuracy of hand-written digit recognition using
+PyTorch distributed data parallel and MNIST. We optimize the neural network architecture as well
+as the optimizer configuration. As it is too time consuming to use the whole MNIST dataset, we
+here use a small subset of it.
+
+You can execute this example with mpirun command as follows:
```

Comment:
If we modify this code for GPUs, we might need to install pytorch from source instead of binaries because binary version does not support MPI. So I would like to see some comment to mention it.

Reference: [`MPI Backend` part in this page](https://pytorch.org/tutorials/intermediate/dist_tuto.html)

Reply:
Thank you for your suggestion. I added a note for the MPI backend in the commit https://github.com/optuna/optuna/pull/2303/commits/3f153b273789186b32e6d559e493ad0e5480e45c.

In my environment, the following error occurred if I switched the backend without GPUs. 

```console
RuntimeError: Distributed package doesn't have MPI built in. MPI is only included if you build PyTorch from source on a host that has MPI installed.
```

So, I guess the error does not depend on the usage of GPUs, but it only depends on the MPI backend.

Reply:
Awesome! 

I have tested it on GPUs, but I think so.